api-extractor/src/definitions/ApiMethod.ts

```diff
@@ -26,15 +26,26 @@ export default class ApiMethod extends ApiMember {
       this.params = [];
       for (const param of methodDeclaration.parameters) {
         const declarationSymbol: ts.Symbol = TypeScriptHelpers.tryGetSymbolForDeclaration(param);
-        this.params.push(new ApiParameter({
+        const apiParameter: ApiParameter = new ApiParameter({
           extractor: this.extractor,
           declaration: param,
           declarationSymbol: declarationSymbol,
           jsdocNode: param
-        }));
+        });
+
+        this.innerItems.push(apiParameter);
+        this.params.push(apiParameter);
       }
     }
 
-    this.returnType = methodDeclaration.type ? methodDeclaration.type.getText() : '';
+    // Return type
+    if (!(this.name === '__constructor')) {
```

Comment:
This seems weird to me.  I would expect to see something like:

```
if (this.kind === ApiKind.Constructor)
```
or
```
if (this is ApiConstructor)
```
or
```
if (this.isConstructor) 
```

However, it's out of scope for this PR.  Fix it up in some future PR.